index.wsgi

```diff
@@ -0,0 +1,44 @@
+# -*- coding: utf-8 -*-
+# vim: ft=python:
+# @module index.wsgi
+# @desc Django wsgi start script
+# @since 2024.03.06, 20:47
+# @changed 2024.03.06, 20:47
+
+import os
+import sys
+import time
+import traceback
+import signal
+from pathlib import Path
+
+# App root path
+rootPath = os.path.dirname(os.path.abspath(__file__))
+
+# Detect home path...
+home = str(Path.home())
+
+# TODO: Get the venv path from config?
+activate_this = home + '/.virtualenv/bin/activate_this.py'
+with open(activate_this) as f:
+    code = compile(f.read(), activate_this, 'exec')
+    exec(code, dict(__file__=activate_this))
+
+# Inject application path...
+sys.path.insert(1, rootPath)
+
+import django
+
+if django.VERSION[1] <= 6 and django.VERSION[0] <= 1:
```

Comment:
We are Django 5+ - why do we need code here for 1.6?

Reply:
> We are Django 5+ - why do we need code here for 1.6?

Ok, just removed it. But I'm not sure that this file is required at all: I use it only to test the app on my demo server.

And, if we'll use it, then we'll need at least to add configuration to allow choosing the proper venv name/location or to don't use it at all.